src/main.cpp

```diff
@@ -469,6 +471,8 @@ void printHelp() {
     "\n"
     "    --mavlink-port <port>  - UDP port for mavlink telemetry        (Default: 14550)\n"
     "\n"
+    "    --mavlink-rec-on-arm   - Start recording when armed\n"
```

Comment:
I wonder if `--dvr-on-arm` would be a better name? Or at least `--mavlink-dvr-on-arm`?

Reply:
I like the `dvr` over `rec` idea.
I would vote for `--mavlink-dvr-on-arm` to underline that this will listen on mavlink.
Maybe there will be more status sources in the future e.g. msp, elrs, elrs-backpack, px4.

Reply:
Ok, let's have `--mavlink-dvr-on-arm`